File: frontend/lib/features/accounting/domain/entities/trial_balance_row.dart
class TrialBalanceRow {
  const TrialBalanceRow({
    required this.accountId,
    required this.accountCode,
    required this.accountName,
    required this.category,
    required this.normalBalance,
    required this.debitAmount,
    required this.creditAmount,
    required this.balanceAmount,
    required this.balanceDirection,
  });

  final String accountId;
  final String accountCode;
  final String accountName;
  final String category;
  final String normalBalance;
  final double debitAmount;
  final double creditAmount;
  final double balanceAmount;
  final String balanceDirection;
}

File: backend/src/main/java/com/microflow/accounting/domain/model/TrialBalanceRow.java
package com.microflow.accounting.domain.model;

import java.math.BigDecimal;

public record TrialBalanceRow(
        String accountId,
        String accountCode,
        String accountName,
        String category,
        String normalBalance,
        BigDecimal debitAmount,
        BigDecimal creditAmount,
        BigDecimal balanceAmount,
        String balanceDirection
) {
}
